### ISSUE TITLE
Honor GCT header when parsing dataset rows

### ISSUE THREAD

**Issue by ACastanza** [GSEA-MSigDB/gsea-desktop#51]
Currently GSEA Desktop will parse the GCT header and then throw an error if there are additional blank rows at the end of the gct file. I think that the correct behavior here would be to honor the header and exclude blank rows that occur after the end of the specified number of genes perhaps providing a warning message instead.

**Comment by davideby** [comment]
Yeah, I tend to agree.  GCT was designed in a time of memory-constrained computers where it was more important to know how many items it should expect than to be more user-friendly.  We can clearly change the focus there.

Skipping blank rows is simple thing.

I'm just not totally clear what to do if there are more "real" rows than specified.  I would say that should still be an error.

Honestly we should just steer people to the TXT format since GCT doesn't bring much to the table at this point.
